### PR TITLE
cleaned up namespaces

### PR DESCRIPTION
**Pull request by enikao**
implements #43

### CODE REVIEW

File: src/LionWeb.Core.Generator/LionWeb.Core.Generator.csproj [hunk 1]
```diff
@@ -6,13 +6,13 @@
         <TargetFramework>net8.0</TargetFramework>
         <PackageReadmeFile>README.md</PackageReadmeFile>
         <PackageTags>LionWeb</PackageTags>
-        <RootNamespace>LionWeb_CSharp_Generator</RootNamespace>
+        <RootNamespace>LionWeb.Core.Generator</RootNamespace>
```
Comment (enikao) [inline src/LionWeb.Core.Generator/LionWeb.Core.Generator.csproj:9]: `LionWeb.Core.Generator` or `LionWeb.Generator`?

Comment (dslmeinte) [reply src/LionWeb.Core.Generator/LionWeb.Core.Generator.csproj:9]: `LionWeb.Generator`, as you wouldn't include these classes for core use of LionWeb.

**Comment by dslmeinte** [comment]
Ideally, there's a top-level (e.g. the `README` at root level?) description of the various namespaces, and what can be found (and should go) in there.

**Review by dslmeinte — APPROVED** [review]
Ideally, there's a top-level (e.g. the README at root level?) description of the various namespaces, and what can be found (and should go) in there.

**Comment by enikao** [comment]
> Ideally, there's a top-level (e.g. the README at root level?) description of the various namespaces, and what can be found (and should go) in there.

added / updated

**Pull request merged**